### PR TITLE
Currency converter in React

### LINKED ISSUE #112: Android currency converter
# I'm submitting a
 [] feature request.

****Proposed Feature:****
An android app that will Convert between different currency formats.

****Do you want to work on this issue?****

Yes, I want to work over this issue in Android, under SWoC 2021

### PR DESCRIPTION
**Pull request by pritamp17**
Fixes #112

### Checklist:
- [x] My branch is up-to-date with upstream/master branch.

### Description:
App through which you can compare two different country's currencies.

### Snapshot:
currency2

### CODE REVIEW

**Comment by Tejas1510** [comment]
@pritamp17 please stash all your commit

File: Reactjs/Currency-converter/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+The page will reload if you make edits.<br />
+You will also see any lint errors in the console.
+
+### DEMO
+
+![Alt text](https://github.com/pritamp17/Hacking-Scripts/blob/currency_converter/Reactjs/Currency-converter/currency2.png?raw=true )
+
+
+
```
Comment (atarax665) [inline Reactjs/Currency-converter/README.md:25]: Remove these extra blank lines.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by pritamp17** [comment]
sir, I have made changes in the readme, but I am facing difficulty in stash all commits. could you please help me?

**Comment by Tejas1510** [comment]
@pritamp17 you can have a look at https://stackoverflow.com/questions/26884364/how-to-stash-my-previous-commit
@atarax665 @tripathyprateek Can also guide you more if you have doubt about stashing all commits

**Pull request merged**